deploy/kubernetes-1.13/pmem-app.yaml

```diff
@@ -0,0 +1 @@
+../kubernetes-1.11/pmem-app.yaml
```

Comment:
Can you squash this commit with the corresponding code change?

My goal is to look at the PR commit by commit (it's too large otherwise), and without this change here the code change is incomplete.

You also need change the documentation in that same commit and explain that Kubernetes >= 1.13 is needed.

Remove the `deploy/kubernetes-1.11` and `deploy/kubernetes-1.12` directories.

Reply:
Done